cmd/state/autoupdate.go

```diff
@@ -19,17 +22,35 @@ import (
 	"github.com/ActiveState/cli/internal/osutils"
 	"github.com/ActiveState/cli/internal/output"
 	"github.com/ActiveState/cli/internal/profile"
+	"github.com/ActiveState/cli/internal/rtutils/p"
 	"github.com/ActiveState/cli/internal/updater"
 	"github.com/thoas/go-funk"
 )
 
-const CfgKeyLastCheck = "auto_update_lastcheck"
+const (
+	CfgKeyLastCheck = "auto_update_lastcheck"
+
+	// Ananlytics labels
+	anaLabelSuccess         = "success"
+	anaLabelFailed          = "failed"
+	anaLabelTrue            = "true"
+	anaLabelForward         = "forward"
+	anaLabelUnitTest        = "unittest"
+	anaLabelConflict        = "conflict"
+	anaLabelDisabledEnv     = "disabled_env"
+	anaLabelDisabledConfig  = "disabled_config"
+	anaLabelCI              = "ci"
+	anaLabelFreshInstall    = "fresh-install"
+	anaLabelLocked          = "locked"
+	anaLabelTooFreq         = "too-frequent"
+	anaLabelSpecificVersion = "specific-version"
```

Comment:
These aren't in the doc but I added them as there are error cases for them. See below where they are used.